src/test/regress/expected/multi_extension.out

```diff
@@ -1792,6 +1792,7 @@ SELECT citus_remove_node('localhost', :master_port);
 -- confirm that we can create a tenant schema / table on an empty node
 SET citus.enable_schema_based_sharding TO ON;
 CREATE SCHEMA tenant_schema;
+NOTICE:  distributing the schema tenant_schema
```

Comment:
do we want to introduce this notice? (not sure)

Reply:
Both `citus_schema_distribute` and `citus_schema_undistribute` implies a long operation. Maybe I can remove notices from both of them. What do you think?

Reply:
Preferably, we should only notice them when we use them in indirect way.